pkg/signup/service/signup_service.go

```diff
@@ -383,8 +384,7 @@ func (s *ServiceImpl) DoGetSignup(ctx *gin.Context, cl namespaced.Client, userna
 	} else if completeCondition.Reason == toolchainv1alpha1.UserSignupUserBannedReason {
 		log.Info(nil, fmt.Sprintf("usersignup: %s is banned", userSignup.GetName()))
 		// UserSignup is banned, let's return a forbidden error
-		return nil, apierrors.NewForbidden(schema.GroupResource{}, "",
-			errs.New("Access to the Developer Sandbox has been suspended due to suspicious activity or detected abuse."))
+		return nil, apierrors.NewForbidden(schema.GroupResource{}, "", errs.New(BannedUserMessage))
```

Comment:
As before, the error might be pre-created as a package-level var and just returned from here.

Reply:
yup ac0a1b7